cdk-infra/test/cdk-infra.test.ts

```diff
@@ -0,0 +1,15 @@
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import FhirWorksStack from '../lib/cdk-infra-stack';
+
+test('Resources created', () => {
```

Comment:
Is there another ticket for testing?

Reply:
Yep! It will be after the implementation guides are added.